build/changelog/ChangelogBuilder.php

```diff
@@ -95,10 +95,10 @@ private function createTag($changelogFile)
             ++$tag[1];
             $tag[2] = 0;
             return implode(".", $tag);
-        } else {
-            ++$tag[2];
-            return implode(".", $tag);
         }
+
+        ++$tag[2];
+        return implode(".", $tag);
```

Comment:
Could this be refactored to instead keep the else and have a singular return?

```php
        if ($this->newServiceFlag) {
            //Minor Version Bump if a feature is being released
            ++$tag[1];
            $tag[2] = 0;
        } else {
            ++$tag[2];
        }

        return implode(".", $tag);
```

Reply:
Good catch, gonna change it :+1: